gems/pending/util/miq_winrm.rb

```diff
@@ -17,17 +18,33 @@ def connect(options = {})
     validate_options(options)
     @uri        = build_uri
     @connection = raw_connect(@username, @password, @uri)
+    @operations = 0
     @executor   = @connection.create_executor
   end
 
+  def close
+    @executor.close
+    @executor = nil
+  end
```

Comment:
@Fryguy my intent is that after a close a caller wishing to use the MiqWinRM object would have do call MiqWinRM.connect again, which would create a new @executor.   Also in your example above - I assume you want the return from the executor method to be @executor, not executor, correct?